it/lessons/basics/composition.md

```diff
@@ -140,7 +140,7 @@ defmodule Example do
 end
 ```
 
-Se c'è un conflitto tra due alias o vuoi semplicemente rinominare con un nome del tutto differente, possiamo usare l'opzione `:as`:
+Se c'è un conflitto tra due alias o vuoi semplicemente rinominare il modulo con un nome del tutto differente, si può usare l'opzione `:as`:
```

Comment:
"Se c'è un conflitto" => "in caso di conflitto" or "Dovesse verificarsi un conflitto"

"si può" => "puoi" as previously you referred to the user using the personal form "vuoi". Alternatively, I'd convert the first sentence to the impersonal form, using "è possibile" instead of "vuoi"